lms/djangoapps/learner_home/test_views.py

```diff
@@ -852,7 +867,7 @@ def test_waffle_flag_off(self):
         Verify API returns 400 if waffle flag is off.
```

Comment:
Please also change the status code `400` to `404` in the doc string. same for other docstrings.